www_src/components/modal/modal.jsx

```diff
@@ -0,0 +1,50 @@
+var React = require('react');
+var dispatcher = require('../../lib/dispatcher');
+
+var Modal = React.createClass({
+  getInitialState: function () {
+    return {
+      isVisible: false
+    };
+  },
+  show: function () {
+    this.setState({
+      isVisible: true
+    });
+  },
+  hide: function () {
+    this.setState({
+      isVisible: false
+    });
+  },
+  componentDidMount: function () {
+    dispatcher.on('showModal', (event) => {
```

Comment:
Actually...disregard that. I get it. It'll be needed once the modal's content isn't static... :+1: